### PR TITLE
#49 add rule for finding unencrypted username in rds instance

### PR DESCRIPTION
**Pull request by xysr89**
https://github.com/stelligent/cfn_nag/issues/49

added rule for username check, along with a few minor changes

- renamed the user password rule context to include `Default credentials are not recommended`
- removed a comma in [json/rds_instance/rds_instance_without_publicly_accessible.json](https://github.com/stelligent/cfn_nag/compare/master...xysr89:master#diff-edd3c5aff1e17e1fda1654aa4b68dacb) even it is not a valid json it still passes rspec <span class="emoji-outer emoji-sizer"><span class="emoji-inner" style="background: url(chrome-extension://immhpnclomdloikkpcefncmfgjbkojmh/emoji-data/sheet_apple_32.png);background-position:46.00470035252644% 6.0517038777908345%;background-size:5418.75% 5418.75%" data-codepoints="1f47b"></span></span>👻

### CODE REVIEW

File: spec/cfn_nag_integration/cfn_nag_rds_instance_spec.rb [hunk 1]
```diff
@@ -30,7 +30,60 @@
...
+    it 'flags a violation' do
+      template_name = 'json/rds_instance/rds_instance_no_echo_with_default_password.json'
+
+      expected_aggregate_results = [
+        {
+          filename: test_template_path(template_name),
+          file_results: {
+            failure_count: 2,
+            violations: [
+              Violation.new(id: 'F23',
```
Comment (xysr89) [inline spec/cfn_nag_integration/cfn_nag_rds_instance_spec.rb]: one minor problem i have here is that the violations are not ordered. tested with a  sample template which gives me a similar result. https://gist.github.com/xysr89/41c6d18be6b913fbcff9f2b29dae0d45

not really an issue but its worth to bring it up

**Comment by xysr89** [comment]
@erickascic can this be merged :smiley:

**Comment by erickascic** [comment]
Hey sorry for delay.  I will try to get this merged today.

**Comment by jeffb4** [comment]
Thanks for the patch, Jason. Please run rubocop over branches in the future, I'm doing some lint work on the codebase and it'd be a help.

**Pull request merged**